### ISSUE TITLE
On fresh ubuntu 18.04 package gcsfuse can't be found

### ISSUE THREAD

**Issue by wojciehm** [GoogleCloudPlatform/gcsfuse#394]
Hi,

Unfortunately 20.04 isn't supported so I wanted to test it on 18.04. On fresh install after adding repo install is failing

```
sudo apt-get update
Hit:1 http://de.archive.ubuntu.com/ubuntu bionic InRelease
Hit:2 http://de.archive.ubuntu.com/ubuntu bionic-updates InRelease
Hit:3 http://de.archive.ubuntu.com/ubuntu bionic-backports InRelease
Hit:4 http://de.archive.ubuntu.com/ubuntu bionic-security InRelease
Get:5 http://packages.cloud.google.com/apt gcsfuse-bionic InRelease [3,724 B]
Err:5 http://packages.cloud.google.com/apt gcsfuse-bionic InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 6A030B21BA07F4FB
Reading package lists... Done
W: GPG error: http://packages.cloud.google.com/apt gcsfuse-bionic InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 6A030B21BA07F4FB
E: The repository 'http://packages.cloud.google.com/apt gcsfuse-bionic InRelease' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
```

And then
```
sudo apt-get install gcsfuse
Reading package lists... Done
Building dependency tree
Reading state information... Done
E: Unable to locate package gcsfuse
```

Thanks for your help in advance.

**Comment by bjornleffler** [comment]
This is fixed. We released version 0.30.0 on 10 June and also created a new repo for Ubuntu 20.

**Comment by bjornleffler** [comment]
Duplicate of #391
